test/integration/registration/managedcluster_accept_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"open-cluster-management.io/ocm/pkg/registration/hub"
 )
 
-var _ = Describe("ManagedCluster set hubAcceptsClient from true to false", Ordered, Label("managedcluster-accept"), func() {
+var _ = Describe("ManagedCluster set hubAcceptsClient from true to false", Ordered, Label("managedcluster-accept", CloudEventsTestLabel), func() {
```

Comment:
instead of adding this label for cloudevents, I expected more on having a label to skip test when type is grpc